exhaust.lua

```diff
@@ -0,0 +1,16 @@
+local timestamp = {}
+stamina.register_on_exhaust_player(function(player)
+	if not player then return end
+	local now = minetest.get_us_time()
+	if timestamp[player:get_player_name()] then
+		if now - timestamp[player:get_player_name()] > 999999 then
+			local exhaustion = stamina.get_exhaustion(player) or 0
+			if exhaustion >= stamina.settings.exhaust_lvl then
```

Comment:
Seems like this check always fails, not exactly sure what exhaustion actually represesnts. Testing on local server stamina.settings.exhaust_lvl is 160 and stamina.get_exhaustion(player) returns something between 0 and 140.

Reply:
Exhaustion is incremented by some value depending on what player is doing and gets reset when eating (and by some other things I'm not exactly sure about).
Seems like should use `change` here instead.